Misc/NEWS.d/next/Core and Builtins/2023-05-24-12-10-54.gh-issue-104690.HX3Jou.rst

```diff
@@ -0,0 +1 @@
+Starting new threads and process creation through fork/vfork at interpreter exit is no longer supported, as it can lead to race condition between main Python thread freeing existing thread states and :func:`_thread.start_new_thread` trying to allocate and use state of just created thread or fork/vfork trying to use main thread state in child process.
```

Comment:
Don't mention internal APIs like `_thread.start_new_thread` in the news entry, just say something like "while the Python threading internals try to allocate ..."